units/INU3004/INU3004_unit.bp

```diff
@@ -64,8 +64,8 @@ UnitBlueprint {
         AirThreatLevel = 0,
         ArmorType = 'Normal',
         EconomyThreatLevel = 0,
-        Health = 1600,
-        MaxHealth = 1600,
+        Health = 800,
```

Comment:
nomads dont have a mobile shields, therefore the amount of hp for the t3 mobile arty has to be higher compared to the other factions

Reply:
ok